packages/browser-wallet/src/popup/popupX/pages/CreateAccount/CreateAccount.tsx

```diff
@@ -10,9 +10,31 @@ import Button from '@popup/popupX/shared/Button';
 import { generatePath, useNavigate } from 'react-router-dom';
 import { absoluteRoutes } from '@popup/popupX/constants/routes';
 
+/**
+ * Get the valid identities, which is Identities that are confirmed by the ID provider and are not
+ * expired.
+ * This is not recomputed by change of current time, meaning the returned identities might become
+ * expired over time.
+ */
+function useValidIdentities(): ConfirmedIdentity[] {
+    const identities = useAtomValue(identitiesAtom);
+    return useMemo(() => {
+        const now = new Date();
+        return identities.flatMap((id) => {
+            if (id.status !== CreationStatus.Confirmed) {
+                return [];
+            }
+            const validToDate = new Date(id.idObject.value.attributeList.validTo);
```

Comment:
This does not seem to work, as the the dates are stored as "YYYYMM" which is not parsed properly by the `Date` constructor. I thought there was a helper function to parse this in `wallet-common-helpers/utils/timeHelpers.ts`, but I cannot seem to find it..

Reply:
ahh right, I forgot this was the format.
I guess my test then just happened then... I'll fix this. Good catch